client/lib/post-normalizer/test/index.js

```diff
@@ -51,6 +19,36 @@ function asyncTransform( post, callback ) {
 }
 
 describe( 'post-normalizer', function() {
+	let normalizer, safeImageUrlFake, allTransforms;
+
+	require( 'lib/react-test-env-setup' ).auto();
+	require( 'test/quick-mock' )( __dirname );
```

Comment:
> How do we tear-down quick mocks between tests?

Right now, you don't. If you have that need, we can add a variant that uses `beforeEach` / `afterEach` instead of `before` / `after`. Then you can just include the call in the describe block you want to mock inside. Might get a little hairy if you have nested mockery things... We'd need to notice that mockery is already on and avoid double enabling it i think?